package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reach/skip-nav": "^0.16.0",
```

Comment:
Everything needs to be a dev dependencies except for serve,  because everything is served from static compiled files in production.

Reply:
I don't understand. Isn't the dependencies section for libraries required for the application to run in production?
It sure looks like react and mui are still in the production JS bundles currently. Were they supposed to be stripped out from the production JS bundle generated?

Reply:
The app is compiled to static files so only what is needed from the dependency are included.

Only thing to serve them in production is the “serve” library.

Reset the package and package lock files and install it again but with the flag ‘’’—save-dev’’’